docs/conf.py

```diff
@@ -109,6 +110,9 @@
 release = version
 
 
+_is_readthedocs = bool(os.getenv("READTHEDOCS"))
```

Comment:
```suggestion
_IS_READTHEDOCS = bool(os.getenv("READTHEDOCS"))
```
and everywhere that uses the variable

Reply:
https://github.com/DisnakeDev/disnake/pull/859/commits/3b1624be1a14325f63753b362e9bcf82f36a8820